test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/CheckAllAnswersAndSubmitControllerSpec.scala

```diff
@@ -401,3 +406,5 @@ class CheckAllAnswersAndSubmitControllerSpec
   }
 
 }
+
+object CheckAllAnswersAndSubmitControllerSpec extends CheckAllAnswersAndSubmitControllerSpec {}
```

Comment:
I'm guessing you don't need this object here

Reply:
oh - you could reuse the pattern we had with the other controller specs where you move the `validateAllCheckYourAnswersSections` method into the companion object instead of extending the whole spec

Reply:
fixed